### PR TITLE
Make OTA Engine Enabled By Default

### PR DESCRIPTION
**Pull request by scooterthedev**
Enables the OTA engine by default. Trust me, last PR lol.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 *Welcome to X1Plus!*  If you're a user, you probably want to [go straight to
 the wiki](https://github.com/X1Plus/X1Plus/wiki), which tells you everything
-you need to know about installing and using X1Plus on your printer.
+you need to know about installing and using X1Plus on your prin ter.
```
Comment (riptidewave93) [inline README.md:7]: I feel like this was an accident.

File: bbl_screen-patch/patches/printerui/qml/settings/X1PlusOTADialog.qml [hunk 2]
```diff
@@ -11,7 +11,7 @@ Item {
     property alias name: textConfirm.objectName
     property var currentVersion: screenSaver.cfwVersions['cfw']['version']
     property var ota: X1Plus.OTA.status()
-    property var otaEnabled: !!X1Plus.Settings.get("ota.enabled", false)
+    property var otaEnabled: !!X1Plus.Settings.get("ota.enabled", True)
```
Comment (riptidewave93) [inline bbl_screen-patch/patches/printerui/qml/settings/X1PlusOTADialog.qml:14]: Pretty sure we need this lowercase, similar to false. @jwise can confirm, but lots of languages are very picky on boolean value spelling.

File: README.md (hunk 1 above)
Comment (scooterthedev) [reply README.md:7]: woops, sorry. It's been a long day.

**Comment by jwise** [comment]
Is this PR tested?  I don't believe it has been.  I see evidence that this would not work.

**Comment by scooterthedev** [comment]
> Is this PR tested? I don't believe it has been. I see evidence that this would not work.

Not yet. I think for the settings.py, I believe the syntax is uppercase because that is the formatting for the Boolean. The rest of the files I changed to lowercase.

**Comment by jwise** [comment]
Please do not submit PRs that have not been tested.  In this case, there is at least one change that, if merged, will cause the OTA subsystem to entirely fail to initialize.

**Comment by scooterthedev** [comment]
> Please do not submit PRs that have not been tested. In this case, there is at least one change that, if merged, will cause the OTA subsystem to entirely fail to initialize.

Got it. Sorry about that. I will test tomorrow if you want.

**Review by riptidewave93 — APPROVED** [review]
Visually this looks good, but no merging until it is tested. As of this commit, I approve of the PR as long as testing of these changes works as expected.

**Comment by jwise** [comment]
dup #298

**Pull request closed without merging**